chart/values.yaml

```diff
@@ -446,6 +446,12 @@ ingress:
   #host: ""
   cert_email: "test@example.com"
   tls: false
+
+  # This is related to the issues webrecorder/browsertrix#1570 and cert-manager/cert-manager#6184
+  # This relates to the new `http01.ingress.ingressClassName` solver spec attribute which got introduced in the recent v1.15 cert-manager version.
+  # Cert-manager v1.15 does not want the fields ingressClassName and class be set at the same time.
+  includeClassAnno: false
```

Comment:
I wonder if this should be called something else, like `useOldClassField` instead of `includeClassAnno` since its not an annotation and its specifically about the old field?